quadrat/sass/blocks/_post-template.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-post-template {
```

Comment:
I think I understand why this is necessary, because Gutenberg outputs this block as a `ul` and the browser styles add a margin to these by default...

Reply:
Yeah that's right, I was just overriding the browser defaults and getting rid of the extra space it was causing.